docs/infrastructure-metallb.md

```diff
@@ -28,14 +28,7 @@ kubectl apply -f /etc/genestack/manifests/metallb/metallb-namespace.yaml
 
 !!! tip
 
-    It is recommended that you modify the file locally so that your changes are not impacted by the git tree.
```

Comment:
we should keep the tip here which instructs the user to create the file or create docs letting the user know what needs to be done.